src/models/data.py

```diff
@@ -180,6 +180,32 @@ def calculate_normalizing_array(self, data_vars: List[str]):
             'std': np.vstack(std).swapaxes(0, 1)
         })
 
+    @staticmethod
+    def move_target_var_to_end_of_dataset(x: xr.Dataset, y: xr.Dataset) -> xr.Dataset:
```

Comment:
I added this because it allows us to be sure that the final array (index = `-1`) of the numpy array is the target variable when we subselect the `current` month data for all of the variables we are interested in

Reply:
Couldn't you just select the indices using x.data_vars?

in line 226:
```python
relevant_indices = [idx for idx, feat in enumerate(x.data_vars) if feat != target_variable]
current = x_np[:, relevant_indices, -1]
```

Reply:
i can't figure out why but this doesn't seem to work in the code htat i'm testing?

Reply:
i messed up the indexing

Reply:
i guess that's one reason why i like the xarray approach. there must be a more elegant way of doing what i am doing here

Reply:
What's not working about this?
what error do you get? If we can replace an entire function with a line of code, we should try

Reply:
Saw its fixed; nicely done!